plugin/hpsa/hpsa.py

```diff
@@ -502,12 +503,26 @@ def _hp_disk_to_lsm_disk(hp_disk, sys_id, ctrl_num, key_name,
         disk_type = _disk_type_of(hp_disk)
         blk_size = int(hp_disk['Native Block Size'])
         blk_count = int(_hp_size_to_lsm(hp_disk['Size']) / blk_size)
+
+        if 'Disk Name' in hp_disk.keys():
+            regex_match = re.compile("/dev/(sd[a-z]+)").search(disk_sd_path)
```

Comment:
```disk_sd_path``` doesn't appear anywhere else, I think you must mean something else.

Also, I can't help thinking that the pyudev library would be the right thing to use here.